packages/castmill/lib/castmill_web/live/admin/resources.ex

```diff
@@ -22,7 +22,7 @@ defmodule CastmillWeb.Live.Admin.Resources do
       |> assign(:options, %{
         page: 1,
         page_size: 10,
-        search: "",
+        search: ""
```

Comment:
Trailing commas are actually a good thing, as when adding more items it will only create a diff per new item, without the trailing comma it will also diff on the previous line.

Reply:
Agree but it doesn't seem to be configurable. Mix format tries to mandate a single style and doesn't provide a lot of configuration options. A pity that they didn't include this in the "Elixir style"...

> Our goal with the formatter is to mandate a style, reducing the amount of
> discussion on each project or change and also ensuring everyone in the
> community follows the same guidelines.

Reply:
It seems that in Elixir sometimes trailing commas are not correct syntactically, so that may be why it is not supported:
https://elixirforum.com/t/do-you-always-add-a-trailing-comma/21156/4

Reply:
Sounds reasonable